components/Navbar.tsx

```diff
@@ -41,32 +42,42 @@ export function Navbar() {
 		<header className='p-4 flex-[0.25] fixed top-0 z-50 w-screen'>
 			<nav
 				className={`${
-					navbar ? 'max-h-screen' : 'max-h-16'
+					navbar ? 'max-h-screen' : 'max-h-20'
 				} bg-dark-600/50 p-4 rounded-md max-w-5xl mx-auto backdrop-blur-lg transition-all duration-500 ease-in-out overflow-hidden`}
 			>
 				<div className='flex flex-row justify-between items-center'>
-					<div>Logo</div>
-					<div className='sm:hidden text-light-500 text-2xl h-6'>
+					<div className='flex flex-row items-center gap-x-2'>
+						<Link href='/'>
+							<Image
+							src='/city-cyclery-logo-white.svg'
+							alt='City Cyclery Logo'
+							width={60}
```

Comment:
Maybe make this a bit smaller, I'm not quite kept up on what's happening about the assets haha